src/schema/zoning-district.ts

```diff
@@ -1,9 +1,15 @@
 import { pgTable, text, uuid } from "drizzle-orm/pg-core";
 import { multiPolygonGeog, multiPolygonGeom } from "../drizzle-pgis";
+import { createSelectSchema } from "drizzle-zod";
+import { z } from "zod";
 
 export const zoningDistrict = pgTable("zoning_district", {
   id: uuid("id").defaultRandom().primaryKey(),
   label: text("label").notNull(),
   wgs84: multiPolygonGeog("wgs84", 4326).notNull(),
   liFt: multiPolygonGeom("li_ft", 2263).notNull(),
 });
+
+export const selectZoningDistrictSchema = createSelectSchema(zoningDistrict);
+
+export type SelectZoningDistrict = z.infer<typeof selectZoningDistrictSchema>;
```

Comment:
Not going to hold up approval but I think I prefer just `ZoningDistrict` for this type, but I know Drizzle has docs showing both naming patterns.

Reply:
I go back and forth in my head. For sanity's sake, it's probably best to stick with it as-is. If it becomes bothersome, we can always refactor to the other style. (Also, it's easier to search and replace on `SelectZoningDistrict` than just `ZoningDistrict`)